common/src/main/java/org/jboss/da/common/util/FileUtils.java

```diff
@@ -0,0 +1,25 @@
+package org.jboss.da.common.util;
```

Comment:
We already have apache commons io as dependency in our root pom.xml which provides the `deleteDirectory` method.